brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaFacade.java

```diff
@@ -152,4 +123,7 @@
      * @return the resolved property specified by key
      */
     Object resolveProperty(String nodeId, A toscaApplication, String key);
+
+    Optional<Object> getRelationshipScript(String opKey, A toscaApplication, ToscaApplication.Relationship relationship, String computeName, String expandedFolder);
```

Comment:
We did discuss that, but it kinda seems pointless to put javadoc to say "opKey: the key name of the operation", "toscaApplication: the tosca application". There's not really anything more that can meaningfully be added